src/test/java/com/epam/ta/reportportal/dao/IntegrationTypeRepositoryTest.java

```diff
@@ -30,7 +29,7 @@
 /**
  * @author <a href="mailto:ivan_budayeu@epam.com">Ivan Budayeu</a>
  */
-@Sql("/db/fill/integration/integrations-fill.sql")
+//@Sql("/db/fill/integration/integrations-fill.sql")
```

Comment:
whats wrong?

Reply:
no need anymore. integration types filled in default data script